source/developer/core-developer-handbook.md

```diff
@@ -39,6 +39,15 @@ Below is the list of core developers working on Mattermost:
 - **Jonathan Fritz**
   - @jonathan on [pre-release.mattermost.com](https://pre-release.mattermost.com/core/messages/@jonathan) and [@MusikPolice](https://github.com/MusikPolice) on GitHub
   - Dev areas: Desktop
+- **Martin Kraft**
+  - @martin.kraft on [pre-release.mattermost.com](https://pre-release.mattermost.com/core/messages/@martin.kraft) and [@mkraft](https://github.com/mkraft) on GitHub
+  - Dev areas: All areas
```

Comment:
@saturninoabril Is there any specific area Martin is concentrating on? Like perf measuring / RN apps / permissions?

Reply:
I'm on the permissions team. I have thus far done work in everything except desktop.

Reply:
@jasonblais Not sure about it. I just collected individual inputs on dev areas.

Reply:
@mkraft Any specific area you're concentrating on?

The dev areas will be used by PMs to assign PR reviewers, among other things.

Reply:
@jasonblais No. You can leave it as-is, blank, or choose some areas that no one else wants.